.github/workflows/codeql-analysis.yml

```
@@ -4,8 +4,15 @@ on:
   schedule:
     - cron: '0 10 * * MON'
 
+permissions:
+  contents: read
+
 jobs:
   'Analyze':
+    permissions:
+      actions: read  # for github/codeql-action/init to get workflow details
+      contents: read  # for actions/checkout to fetch code
+      security-events: write  # for github/codeql-action/analyze to upload SARIF results
```

```suggestion
      actions: read # for github/codeql-action/init to get workflow details
      contents: read # for actions/checkout to fetch code
      security-events: write # for github/codeql-action/analyze to upload SARIF results
```

For some reason, the double space before the comment break the linting job. Not sure why, we should fix that on our end, but for the moment, would you mind applying those change. Thanks